src/sentry/relay/config/__init__.py

```diff
@@ -200,6 +207,11 @@ def _get_project_config(project, full_config=True, project_keys=None):
 
             cfg["config"]["dynamicSampling"] = {"rules": active_rules}
 
+    # In this case we should override old conditionnal rules if they exists
+    # or just return uniform rule
+    if allow_dynamic_sampling_basic:
```

Comment:
Should this be an `elif`?

Reply:
no we want to override previous rules, so like remove all condition rules on the fly and return only single uniform rule.

Reply:
Yep agreed here

Reply:
Wouldn't it make sense to put this block _before_ line 187 then? No need to check the feature `organizations:server-side-sampling`, fetch the option `sentry:dynamic_sampling` and call `get_latest_release` if it's going to be overwritten anyway.

So I'm suggesting

```python
if allow_dynamic_sampling_basic:
   ...
elif features.has(
    "organizations:server-side-sampling",
    project.organization,
):
    ...

Reply:
yeah good call, i'll fix.